File: app/src/main/java/com/prashant/masterbuddy/ws/model/RegisterRequest.java
package com.prashant.masterbuddy.ws.model;

import com.google.gson.annotations.SerializedName;

/**
 * Created by tanmay.agnihotri on 5/1/18.
 */

public class RegisterRequest {

    @SerializedName("Name")
    private String name;

    @SerializedName("Email")
    private String EmailId;

    @SerializedName("Password")
    private String password;

    @SerializedName("RegistrationDate")
    private String RegistrationDate;

    public RegisterRequest(String name, String emailId, String password, String registrationDate) {
        this.name = name;
        EmailId = emailId;
        this.password = password;
        RegistrationDate = registrationDate;
    }

    public void setName(String name) {
        this.name = name;
    }

    public void setEmailId(String emailId) {
        EmailId = emailId;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    public void setRegistrationDate(String registrationDate) {
        RegistrationDate = registrationDate;
    }
}

File: app/src/main/java/com/prashant/masterbuddy/ws/model/RegisterRequest.kt
package com.prashant.masterbuddy.ws.model

import com.google.gson.annotations.SerializedName

/**
 * Created by tanmay.agnihotri on 5/1/18.
 */

class RegisterRequest(@field:SerializedName("Name")
                      private var name: String?, @field:SerializedName("Email")
                      private var EmailId: String?, @field:SerializedName("Password")
                      private var password: String?, @field:SerializedName("RegistrationDate")
                      private var RegistrationDate: String?) {

    fun setName(name: String) {
        this.name = name
    }

    fun setEmailId(emailId: String) {
        EmailId = emailId
    }

    fun setPassword(password: String) {
        this.password = password
    }

    fun setRegistrationDate(registrationDate: String) {
        RegistrationDate = registrationDate
    }
}
